starboard/shared/ffmpeg/ffmpeg_dynamic_load_dispatch_impl.cc

```diff
@@ -247,39 +247,55 @@ bool FFMPEGDispatchImpl::OpenLibraries() {
 
 void FFMPEGDispatchImpl::LoadSymbols() {
   SB_DCHECK(is_valid());
-// Load the desired symbols from the shared libraries. Note: If a symbol is
-// listed as a '.text' entry in the output of 'objdump -T' on the shared
-// library file, then it is directly available from it.
+  // Load the desired symbols from the shared libraries. Note: If a symbol is
+  // listed as a '.text' entry in the output of 'objdump -T' on the shared
+  // library file, then it is directly available from it.
+  char* errstr;
+  errstr = dlerror();
+  if (errstr != NULL) {
+    SB_LOG(INFO) << "LoadSymbols - checking dlerror shows:" << errstr;
+  }
 
 #define INITSYMBOL(library, symbol)                                     \
   ffmpeg_->symbol = reinterpret_cast<decltype(FFMPEGDispatch::symbol)>( \
-      dlsym(library, #symbol));
+      dlsym(library, #symbol));                                         \
+  errstr = dlerror();                                                   \
+  if (errstr != NULL) {                                                 \
+    SB_LOG(INFO) << "Load Symbols ran into error:" << errstr;           \
+  }
 
   // Load symbols from the avutil shared library.
   INITSYMBOL(avutil_, avutil_version);
   SB_DCHECK(ffmpeg_->avutil_version);
+  SB_LOG(INFO) << "Opened avutil  - version is:" << ffmpeg_->avutil_version()
+               << std::endl;
```

Comment:
Should we add a DCHECK / CHECK or at least a warning around maximum/minimum known and supported versions ? I'm guessing there might be a an upcoming version that will have changed its API again and our current code will try to use the API in a certain way that isn't supported. When it inevitably crashes this log message might save a couple hours for someone troubleshooting.

Reply:
I don't think we should see breaking API changes. From what I can see its usually more about when a new API was added. We use quite a small subset of the FFMPEG function calls.